### ISSUE TITLE
Simply redesign input/output to so that output is distinct from rest of the text.

### ISSUE THREAD

**Issue by pawankholiya01** [pawankholiya01/Collaborative-Coding-Platform#5]
Currently output just appears on the screen as part of normal text which can be easily unnoticed so redesign it as in other website so that user notices the output when it arrives, potentially you could use loader gifs/svg's for loading Green/Red Color for compilation success and output generated vs compilation error or server failure.
Simple issue can be solved only in frontend with HTML/CSS mainly and Bit of JavaScript.
Basically a Good Easy issue to be solved for first contribution.

Edit : Good Starting point can be hacker earth interface.

**Comment by sethpoly** [comment]
Hey, would I be able to work on this?
